docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[ml-biases]]
+=== Possible biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. People use {ml} in science, healthcare, business, 
+entertainment, and many other fields. Decisions are made backed by {ml} 
+techniques. In such a setup, it is crucial to keep in mind the possible biases 
+that can corrupt results, endanger fairness, and jeopardize the success of even 
+the best-intended initiative. Some of these social biases can make their way to 
+the {ml} analytics results more easily than we might think.
+
+There are various possible biases that can negatively influence the result of a 
+{ml} analysis. *These biases typically are not present in the {ml} algorithms 
+that are used*, but rather exist in the datasets themselves or in the way the 
+results are interpreted. We take a look at the two main types of biases: data 
+related ones and interpretation related ones and recommend some basic practices 
+to avoid them. However, it is still important to remember that the problems 
+thematized in this piece are not solved yet. In fact, they are only a part of a 
+greater question about the ethics behind technology.
+
+
+[float]
+==== Biases in datasets
+
+Built-in biases are inserted in the data that we use to train the {ml} model. 
+For example, it could be the result of manual labeling when the dataset has been 
+annotated with the "ground truth" based on preconception or prejudice. The model 
```

Comment:
> For example ... based on preconception or prejudice.

I think we should assume good intentions.  Maybe soften it like this?

> When we use supervised {ml} techniques, we annotate our data with the "ground truth", which is another opportunity to unwittingly introduce preconceptions or prejudice.

Reply:
@lcawl Also thank you for this! I assumed that preconception is something that works already unwittingly! It'll be more exact this way.